testing/test_text_layout.c

```diff
@@ -0,0 +1,116 @@
+#include "game/gui/text/text_layout.h"
+#include "resources/fonts.h"
+#include "utils/allocator.h"
+#include "utils/str.h"
+#include "video/surface.h"
+#include <CUnit/CUnit.h>
+
+void create_fake_font(font *font) {
+    font_create(font);
+    font->w = 8;
+    font->h = 8;
+    font->size = FONT_BIG;
+
+    // Characters up to and including ' ' and 'A' are 8x8
+    surface s;
+    for(int i = 0; i <= 33; i++) {
+        surface_create(&s, 8, 8);
+        vector_append(&font->surfaces, &s);
+    }
+
+    // 'B' is 4x8
+    surface_create(&s, 4, 4);
+    vector_append(&font->surfaces, &s);
```

Comment:
Please use an active voice in these comments, eg:
```
// Make 'B' 4x8
```

Also: is B 4x8 or 4x4? If the former, please comment as how a 4x4 surface in a 8x8 font makes 'B' an 4x8 character.

Reply:
Fonts no longer have sizes, glyphs do. The font size is only there for compatibility for now... Maybe I will add a comment on that in the test.

Reply:
Btw, what is this active voice thing ? Does this have something to do with english language or tests conventions or what ?

Reply:
the `'B' is 4x8` comment states it as a fact, but not *why* or *when* it is true.

Most of the time, 'B' is not 4x8. It is, to my knowledge, only 4x8 in this test case.

By using a passive voice, the "who" of an action is not present-- here, the "who" is create_fake_font.
It's unfortunate that my suggestion was not in active voice, but instead imperative voice.. yeah.

some examples:
> The linens are ironed. (passive)
> Mr. B ironed the linens. (active)
> Iron the linens. (imperative)

Reply:
Aaha. I usually consider the "who" to be the test, as I write the comments in the test context (test file). But perhaps that's just in my head then. In any case, it's changed.